spec/graphql/schema/resolver_spec.rb

```diff
@@ -509,6 +527,14 @@ def add_error_assertions(field_name, description)
         # (5 + 8) * 3
         assert_equal 39, res["data"]["prepResolver9"]["value"]
       end
+
+      it "supports loading array of ids" do
+        res = exec_query('{ prepResolver9Array(intIds: ["1", "10", "100"]) { value } }')
+        # (1 + 8) * 3
+        # (10 + 8) * 3
+        # (100 + 8) * 3
+        assert_equal [27, 54, 324], res["data"]["prepResolver9Array"].map { |v| v["value"] }
```

Comment:
I'm not a huge fan of the tests in this file because I think some are a bit unclear (including this one), but I added it anyway.

I trust the tests in `spec/graphql/schema/relay_classic_mutation_spec.rb` a bit more. :smile:

Reply:
😅